src/styles/globals.scss

```diff
@@ -59,14 +62,14 @@ $lime: #bfd349;
 $royal-blue: #162ca7;
 $purpleblue: #40537f;
 $lt-grey: #c4c4c4;
-$black: #000000de;
+$black: #000000;
 
 //new colors
 $neon-green: #d5ff2e;
 $sky-blue: #1da1f2;
 $dark-cerulean: #064165;
 $burnt-orange: #ff7f35;
-$nero: #141414;
+$nero: #000000de;
```

Comment:
#000000de isn't a valid color code. Not sure why the 'de' got added to the end but I see you copied the code from 'black'. If we're going to name the variables with color names, then we shouldn't be redefining the codes of the variables, we should be using different variables or renaming the variables to match the new colors.